README.md

```diff
@@ -47,24 +71,18 @@ client_secret="your_client_secret" #Replace this value by the client secret of y
 
 sdk = SDK(client_id=client_id, client_secret=client_secret)
 
-# Creation of a sequence
-reg = Register.square(2, prefix="q")
-sequence = Sequence(reg, devices.Chadoq2)
-# You should add channels and variables to your sequence
-# See Pulser documentation for details
-serialized_sequence = sequence.serialize() # Serialize the sequence in json format
+# Set the wanted values for the variables defined in the sequence
+job1 = {"runs": 20, "variables": {"omega_max": 6} }
```

Comment:
Maybe a quick note specifying what `runs` is ?

Reply:
done